.template/addons/semaphore/Dockerfile.web.tt

```diff
@@ -0,0 +1,6 @@
+ARG DOCKER_REGISTRY=<%= DOCKER_REGISTRY %>
```

Comment:
do we need this file, does it relative to semaphore @hoangmirs 🤔

Reply:
It was added when I rebased, removed it